package.json

```diff
@@ -27,8 +27,9 @@
   },
   "devDependencies": {
     "codeclimate-test-reporter": "0.0.4",
+    "gh-release": "^1.1.7",
     "istanbul": "^0.3.13",
-    "semistandard": "^4.0.3",
+    "semistandard": "^4.1.0",
```

Comment:
Nope.

`^` is probably the most prudent approach. I don't think the major version will change unless there are some new rules that start causing errors in existing repositories.

That being said, I'll add this repo to the `semistandard` tests to be sure it passes linting.